src/data/int/basic.lean

```diff
@@ -476,6 +476,9 @@ theorem mod_add_div : ∀ (a b : ℤ), a % b + b * (a / b) = a
 | -[1+ m] (n+1:ℕ) := mod_add_div_aux m n.succ
 | -[1+ m] -[1+ n] := mod_add_div_aux m n.succ
 
+theorem div_add_mod (a b : ℤ) : b * (a / b) + a % b = a :=
+by rw add_comm; exact mod_add_div
```

Comment:
```suggestion
(add_comm _ _).trans (div_add_mod _ _)
```

Reply:
Thanks. I'll try them locally. For some reason it failed to build when I tried before.